apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue

```diff
@@ -217,6 +226,12 @@ const state = reactive({
 
 /* Components */
 const handleChangeToggle = () => {};
+const handleClickSchedule = () => {
+    collectorPageStore.$patch({
+        visibleModal: true,
+    });
+    collectorScheduleStore.setOriginCollector(props.item.origin);
```

Comment:
suggestion: I suggest not to bring the concern of schedule form here. 😄

![image](https://github.com/cloudforet-io/console/assets/26986739/5d92c0fc-bc4d-49e2-b7f4-e22963abaf65)